tests/test_snapshot_download.py

```diff
@@ -144,6 +144,37 @@ def test_download_model_local_only(self):
             )
             self.assertTrue(self.first_commit_hash in storage_folder)  # has expected revision
 
+        # Test with local_dir
+        with SoftTemporaryDirectory() as tmpdir:
+            # first download folder to local_dir
+            snapshot_download(self.repo_id, local_dir=tmpdir)
+            # now load from local_dir
+            storage_folder = snapshot_download(self.repo_id, local_dir=tmpdir, local_files_only=True)
+            self.assertTrue(str(tmpdir) in storage_folder)  # has expected revision
+
+    def test_download_model_offline_mode_not_in_local(self):
```

Comment:
```suggestion
    def test_download_model_to_local_dir_with_offline_mode(self):
```

(otherwise it's quite close to `test_download_model_offline_mode_not_in_local_dir` below)